pkg/master/reconcilers/mastercount.go

```diff
@@ -149,6 +149,10 @@ func (r *masterCountEndpointReconciler) RemoveEndpoints(serviceName string, ip n
 		return err
 	}
 
+	if len(e.Subsets) == 0 {
```

Comment:
is there a test that covers this?

Reply:
See the new sub-test above in lease_test

Reply:
lease_test test EndpointReconciler's implements, this RemoveEndpoints is an implement of masterCountEndpointReconciler, you should test it in controller_test.go